src/atopile/kicad_plugin/common.py

```diff
@@ -77,6 +77,19 @@ def sync_track(
     source_board.Remove(new_track)
     return new_track
 
+# also pull in any silkscreen items
+def sync_drawing(
```

Comment:
Is this where you were thinking of using the `append` function? https://github.com/atopile/atopile/issues/397#issuecomment-2179397447

Reply:
Yea I was just trying to figure out how id get everything to come in that I've added in the modules layout. This change will make it include drawings which is good, but as discussed in discord, any additional footprints may not be captured.

Reply:
That was going to be an alternate implementation, but instead I think this works as intended.